Split/Api/SyncConfig.swift

```diff
@@ -51,5 +51,12 @@ import Foundation
             builderFilters.append(filter)
             return self
         }
+
+        @discardableResult
+        @objc(addFlagSet:)
+        public func addFlagSet(_ filter: SplitFilter) -> SyncConfig.Builder {
```

Comment:
I don't disagree completely with this method but we haven't added it in other client sides SDKs, we're using the existing methods for filters. Maybe worth discussing offline to align.

Reply:
This is a really good catch. That method shouldn't be there. Will remove it. Thanks!!